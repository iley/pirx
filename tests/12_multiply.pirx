func main(): int {
  var x: int;
  var y: int;
  x = 2;
  y = 3;
  printf("x * y = %d\n", x * y);
  printf("x * 5 = %d\n", x * 5);
  printf("x + y * 4 = %d\n", x + y * 4);
  return 0;
}
