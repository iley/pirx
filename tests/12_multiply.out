x * y = 6
x * 5 = 10
x + y * 4 = 14
